style.css

```diff
@@ -0,0 +1,340 @@
+}
+
+a {
+    color: #444444;
+    text-decoration: none;
+}
+
+ul {
+    list-style-type: none;
+    margin: 0;
+    padding: 0;
+}
+
+section {
+    padding: 5em 2em;
+}
+
+body{
+    background: #FAFAFA;
+
+}
+
+/*Class based Styling*/
+.header{
+    background: white;
+    padding: 0.4em 1em 0.54em 1em;
+    border-bottom: 1px solid #DBDBDB;
+}
+
+.contained-header{
+    max-width: 58em;
+    margin: auto;
+}
+
+.search-bar{
+    padding: .1em;
+    background-color: #FAFAFA;
+    border: 1px solid #DBDBDB;
+    width: 23%;
+    height: 1.67em;
+    border-radius: 0.18em;
+}
+
+.search-bar::placeholder{
+    text-align: center;
+    color: #A1A1A1;
+}
+
+.logo{
+    text-decoration: none;
+    color: black;
+
+    font-size: 1.5em;
+    font-family: cursive;
+}
+
+.container{
+    display: flex;
+    place-content: space-between;
+    align-items: center;
+}
+
+.header-sidebar{
+    display: flex;
+    flex-wrap: nowrap;
+    flex-direction: row;
+    align-items: center;
+    font-family: 'Open Sans', sans-serif;
+    font-size: .95em;
+    /*justify-content: center;*/
+}
+
+.header-sidebar-primary{
+    margin-right: 1em;
+    background: var(--primary-color);
+    color: white;
+    padding: .3em 0.8em 0.4em 0.8em;
+    border-radius: .3em;
+    font-weight: 550;
+    display: inline-block;
+}
+
+.header-sidebar-secondary{
+    color: var(--primary-color);
+    font-weight: 550;
+    display: inline-block;
+}
+
+.container-profile{
+    display: flex;
+    width: 80%;
+    padding-top: 2em;
+    padding-bottom: 2em;
+    align-content: center;
+    justify-content: center;
+    margin: auto;
+    margin-bottom: 0.9em;
+    font-size:0.97em;
+}
+.profile-pic-left{
+    height: 10em;
+    width: 10em;
+    border-radius: 50%;
+    display: flex;
+    margin-right: 6em;
+    margin-left: 2.5em;
+}
+
+.profile-section-right{
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    font-family: 'Open Sans', sans-serif;
+    max-width: 39em;
+    font-size: 1.0em;
+}
+
+.inline{
+    display: flex;
+}
+
+.profile-section-username{
+    font-family: 'Open Sans', sans-serif;
+    font-size: 1.7em;
+    /*font-style: normal;normal*/
+    font-weight: 300;
+    margin-right: 0.8em;
+}
+
+.profile-section-follow{
+    background-color: #0595F6;
+    color: white;
+    font-weight: 550;
+    font-size: 1.0em;
+    padding: 0.3em 0.5em 0.2em 0.5em;
+    border-radius: 0.3em;
+    margin-bottom: 0.4em;
+}
+
+.inline-stats{
+    display: flex;
+    width: 60%;
+    justify-content: space-between;
+    margin: 1em 0;
+
+}
+
+.inline-stats-number{
+    font-family: 'Open Sans', sans-serif;
+}
+
+.profile-content{
+    font-family: 'Open Sans', sans-serif;
+    display: flex;
+    flex-direction: column;
+    line-height: 158%;
+
+    /*letter-spacing: 0.02em;*/
+}
+
+.profile-content-heading{
+    font-family: 'Open Sans', sans-serif;
+    font-weight: 600;
+}
+
+.profile-content-link{
+    color: #0F4173
```

Comment:
Please check if we need to use the variable `primary-color` here
Same for profile-content-link2